.github/workflows/build-check.yml

```diff
@@ -0,0 +1,19 @@
+name: Build check
+on:
+  workflow_dispatch:
```

Comment:
We don't need to include it for workflow_dispatch

Reply:
updated